lib/libxdp/tests/Makefile

```diff
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: (GPL-2.0 OR BSD-2-Clause)
+
+USER_TARGETS := test_xsk_refcnt
+USER_LIBS := -lpthread
+
+EXTRA_DEPS +=
+EXTRA_USER_DEPS += test_utils.h
+
+TEST_FILE := ./test-libxdp.sh
+TEST_RUNNER := ./test_runner.sh
+
+XDP_C = ${XDP_TARGETS:=.c}
+XDP_OBJ = ${XDP_C:.c=.o}
```

Comment:
Let's drop the `XDP_*` variables and rules for now since we don't have any BPF files being compiled in the tests directory.